koans/triangle.rb

```diff
@@ -15,6 +15,21 @@
 #
 def triangle(a, b, c)
   # WRITE THIS CODE
+
+
+        raise TriangleError, "Sides must by numbers greater than zero" if (a <= 0) || (b <= 0) || (c <= 0)
+        raise TriangleError, "No two sides can add to be less than or equal to the other side" if (a+b <= c) || (a+c <= b) || (b+c <= a)
+
+
+    if ((a == b) && (a == c) && (b == c))
+      return :equilateral
```

Comment:
You can not use return explicitly in this case, because the method returns the last completed action as a result

Reply:
return was removed